app/assets/javascripts/3dbio_viewer/src/data/PdbPublications.ts

```diff
@@ -0,0 +1,86 @@
+import { Codec, GetType, array, nullType, number, string } from "purify-ts";
+import { maybe } from "./PdbLigands";
+import { PdbPublication } from "../domain/entities/Pdb";
+
+export const publicationsCodec = array(
+    //using so many "maybe" from examples: 7O7Z, 6ABA, 7SUB
+    //but truly I don't want to use them so often. Check with JR
+    //9zzz will return {}
```

Comment:
Indeed, using maybe just as a general safety measure isn't good, we should check which one are really nullable, and get errors when the source is incorrect.